ml_tools/config.py

```diff
@@ -84,3 +84,12 @@ def deep_copy_map_if_key_not_exist(from_map, to_map):
             deep_copy_map_if_key_not_exist(from_map[key], to_map[key])
         elif key not in to_map:
             to_map[key] = from_map[key]
+
+
+def args_to_config(args):
+    """Convert an argparse args dict to a usable Config object."""
+    if args.config_file:
+        conf = Config.read_config_file(args.config_file)
+    else:
+        conf = Config.read_default_config_file()
+    return Config.load_from_map(conf)
```

Comment:
As of #36, Config no longer has `read_config_file`, `read_defualt_config_file` nor `load_from_map`. `Config.load_from_file` is all that's required now (it returns the default if the config file is `None`.